lib/reek/ast/sexp_extensions/arguments.rb

```diff
@@ -93,6 +93,17 @@ def anonymous_splat?
       module ShadowargNode
         include ArgNodeBase
       end
+
+      # Utility methods for :forward_args nodes.
+      # rubocop:disable Naming/ClassAndModuleCamelCase
```

Comment:
I tried to modify extension key retrieving [here](https://github.com/troessner/reek/blob/master/lib/reek/ast/ast_node_class_map.rb#L29), extracting it to method, so `ForwardArgsNode` can be used here instead, but then it will have `UtilityFunction` smell. Maybe it should be extracted to separate class/module.